js/components/common/Disclaimer.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+/**
+ * A React component representing the disclaimer popup
+ */
+export default class Disclaimer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            dontShowDisclaimer: localStorage.getItem('DontShowDisclaimer')
+        }
+    }
+
+    handleClose = () => {
+        let disclaimer = document.getElementById("disclaimerDiv");
+        disclaimer.classList.toggle("dismiss");
+    }
+
+    handleCheck = () => {
+        localStorage.setItem('DontShowDisclaimer', 'true');
+    }
+
+    render() {
+        const timetable = <a href="https://timetable.iit.artsci.utoronto.ca/">Official Timetable</a>;
+        const calendar = <a href="http://calendar.artsci.utoronto.ca/">Calendar</a>;
```

Comment:
It looks like the new permanent URL is `https://artsci.calendar.utoronto.ca/`, so please update to use this. Also, please change "Calendar" to "Academic Calendar".